src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java

```diff
@@ -38,20 +38,36 @@ protected String getPackageLocation() {
     }
 
     @Test
-    public void testMultiCaseClass() throws Exception {
+    public void testMultiCaseClassOne() throws Exception {
```

Comment:
We need to avoid naming tests and inputs like this, please rename all of these according to content/motivation of test inputs and configuration. I would suggest something like `testMultiCaseSmallTalkStyle`

Reply:
done

Reply:
Please address file names as well, as requested above.

Reply:
@mahfouz72 ping